src/cdk/dialog/dialog-container.ts

```diff
@@ -68,7 +68,7 @@ export function throwDialogContentAlreadyAttachedError() {
     '[attr.aria-modal]': '_config.ariaModal',
     '[attr.aria-labelledby]': '_config.ariaLabel ? null : _ariaLabelledByQueue[0]',
     '[attr.aria-label]': '_config.ariaLabel',
-    '[attr.aria-describedby]': '_config.ariaDescribedBy || null',
+    '[attr.aria-describedby]': '_config.ariaLabel ? null : _ariaDescribedByQueue[0]',
```

Comment:
If there are multiple, shouldn't they all be part of the aria-describedby, not just the first one?

Reply:
Reverted this change to remove any changes to aria-describedby.